inference-engine/src/plugin_api/ie_plugin_itt.hpp

```diff
@@ -0,0 +1,32 @@
+//*****************************************************************************
```

Comment:
can we move to `cpp_interfaces`? In this case it's supposed that `ie_plugin_itt.hpp` should be used only for base plugin classes which are located in `cpp_interfaces`, while real plugins like CPU, GPU are supposed to have their own plugin specific files with ITT domains. And this file probably needs to be ignored from docs generation (e.g. we can add to `EXCLUDE_PATTERNS` in Doxyfile), how do you think?

Reply:
Yes, I agree. It's good idea. Plugins should use their own domains. I'll move it.